4-Soru.py

```diff
@@ -0,0 +1,19 @@
+from collections import OrderedDict
+
+def removeDupWithoutOrder(str):

```

Comment:
copy/paste dikkatsizligi 😉 
```suggestion
def removeDupWithOrder(str):

```